index.html

```diff
@@ -50,7 +53,158 @@ <h2 class="greeting greeting-text headline-box">
+              <li class="language">html</li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </section>
+
+        <section class="project project-text">
+          <div class="project-title">
+            <h4 class="project-name">
+              Multi-Post Stories<br />
+              Gain+Glory
+            </h4>
+            <ul class="langues-use">
+              <li class="language ruby">Ruby on rails</li>
+              <li class="language">CSS</li>
+              <li class="language">Java Script</li>
+              <li class="language">html</li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </section>
+
+        <section class="project project-text">
+          <div class="project-title">
+            <h4 class="project-name">
+              Multi-Post Stories<br />
+              Gain+Glory
+            </h4>
+            <ul class="langues-use">
+              <li class="language ruby">Ruby on rails</li>
+              <li class="language">CSS</li>
+              <li class="language">Java Script</li>
+              <li class="language">html</li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </section>
+
+        <section class="project project-text">
+          <div class="project-title">
+            <h4 class="project-name">
+              Multi-Post Stories<br />
+              Gain+Glory
+            </h4>
+            <ul class="langues-use">
+              <li class="language ruby">Ruby on rails</li>
+              <li class="language">CSS</li>
+              <li class="language">Java Script</li>
+              <li class="language">html</li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </section>
+
+        <section class="project project-text">
+          <div class="project-title">
+            <h4 class="project-name">Multi-Post Stories <br />Gain+Glory</h4>
+            <ul class="langues-use">
+              <li class="language ruby">Ruby on rails</li>
+              <li class="language">CSS</li>
+              <li class="language">Java Script</li>
+              <li class="language">html</li>
+            </ul>
+            <button type="button" class="see-project">See Project</button>
+          </div>
+        </section>
+      </article>
+
+      <!--About me section-->
+      <article class="about-me">
+        <h2 class="about-text">About me</h2>
+        <p class="about-me-intro">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <button type="button" class="resume">Get My Resume</button>
+
```

Comment:
Kindly modify the "Get my resume" button to match the design. Check the above comment for more on why you should stick to the given design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/185593221-08f3cac5-2dcc-4e58-8e57-8eb629112ff0.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/185592599-1aef9f24-fb5b-4dc3-9761-75f981d701cd.png)

The design is the text vertically centralized while yours isn't. Its font size doesn't match yours too.